src/System.Private.CoreLib/src/Microsoft/Win32/Win32Native.cs

```diff
@@ -160,26 +160,6 @@ internal unsafe struct OSVERSIONINFOEX
         [DllImport(Interop.Libraries.Kernel32)]
         internal static extern unsafe int WideCharToMultiByte(uint cp, uint flags, char* pwzSource, int cchSource, byte* pbDestBuffer, int cbDestBuffer, IntPtr null1, IntPtr null2);
 
-        [DllImport(Interop.Libraries.Kernel32, CharSet = CharSet.Auto, SetLastError = true, BestFitMapping = false)]
-        internal static extern bool SetEnvironmentVariable(string lpName, string lpValue);
-
-        [DllImport(Interop.Libraries.Kernel32, CharSet = CharSet.Auto, SetLastError = true, BestFitMapping = false)]
-        private static extern unsafe int GetEnvironmentVariable(string lpName, char* lpValue, int size);
-
-        internal static unsafe int GetEnvironmentVariable(string lpName, Span<char> lpValue)
-        {
-            fixed (char* lpValuePtr = &MemoryMarshal.GetReference(lpValue))
-            {
-                return GetEnvironmentVariable(lpName, lpValuePtr, lpValue.Length);
-            }
-        }
-
-        [DllImport(Interop.Libraries.Kernel32, CharSet = CharSet.Unicode)]
-        internal static extern unsafe char* GetEnvironmentStrings();
-
-        [DllImport(Interop.Libraries.Kernel32, CharSet = CharSet.Unicode)]
-        internal static extern unsafe bool FreeEnvironmentStrings(char* pStrings);
-
         [DllImport(Interop.Libraries.Kernel32, CharSet = CharSet.Auto)]
         internal static extern int GetCurrentThreadId();
```

Comment:
Can `ExpandEnvironmentStringsW` be deleted from this file?

Reply:
Yup.